src/components/Menu.vue

```diff
@@ -46,12 +46,16 @@ export default {
 
   computed: {
     facetValues() {
-      const { data = [] } = this.searchStore.getFacetValues(
```

Comment:
@iam4x FYI: default value assignment when destructuring only kicks on `undefined` values.
The helper often returns `null` values, which is the case here so we can't solely rely on this.